scripts/orchestration/creative_pilot_workspace.py

```
@@ -4,15 +4,17 @@
 from __future__ import annotations
 
 import argparse
+import ctypes
 from datetime import datetime, timezone
 import errno
 import json
 import os
 from pathlib import Path
 import secrets
+import shutil
```

Flake8 (7.3.0)

[error] 'shutil' imported but unused

(F401)